Sources/Meta/Visitors/CopyVisitor.swift

```diff
@@ -1,23 +1,28 @@
 import SwiftSyntax
 
-public class CopyVisitor: SyntaxVisitor, CodegenVisitor {
+public class CopyVisitor: NestedDeclarationVisitor, CodegenVisitor {
     public private(set) var generatedCode = ""
     
     override public func visit(_ node: StructDeclSyntax) -> SyntaxVisitorContinueKind {
-        let structName = node.identifier.description.trimmingCharacters(in: .whitespacesAndNewlines)
+        let visitorContinue = super.visit(node)
```

Comment:
Does it matter if we move this line to `return super.visit(node)`?

Reply:
Yep, we need to invoke `super.init` at the first time, so if we do in the return time, it will be in the last :(

Reply:
Old comment! It changes in this PR